pkg/vulnsrc/redhat-oval/types.go

```diff
@@ -160,7 +160,7 @@ type Definition struct {
 type Entry struct {
 	FixedVersion string `json:",omitempty"`
 	Cves         []CveEntry
-	Arches       []string `json:"Arch,omitempty"`
+	Arches       []string `json:"Arches,omitempty"`
```

Comment:
You can remove `Arches`, then Go uses the same name as the field name.

```suggestion
	Arches       []string `json:",omitempty"`
```

Reply:
looks good, thanks!